packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -229,7 +230,7 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
       // [0.8, 1.0] to [0.0, 1.0], so 0% to 20% of overscroll will produce
       // values for the thumb that range between minLength and the smallest
       // possible value, minOverscrollLength.
-      : safeMinLength * ((fractionVisible - 0.8).clamp(0.0, 0.2) / 0.2);
+      : safeMinLength * ((1.0 - fractionOverscrolled.clamp(0.0, 0.2) / 0.2));
```

Comment:
@LongCatIsLooong Does this look like the right calculation?  I saw you had a PR that simplified this recently (https://github.com/flutter/flutter/pull/31763).

Reply:
Yeah using `fractionOverscrolled` makes a lot more sense!

Reply:
Yeah haha that plus the paragraph comment above.  I think this is consistent with the comment, but I'l keep an eye out for cases where it seems off.